library/src/window_impl.cxx

```diff
@@ -406,8 +407,35 @@ void window_impl::UpdateDynamicOptions()
   renderer->SetRaytracingSamples(opt.render.raytracing.samples);
   renderer->SetUseRaytracingDenoiser(opt.render.raytracing.denoise);
 
+  vtkF3DRenderer::AntiAliasingMode aaMode = vtkF3DRenderer::AntiAliasingMode::NONE;
+
+  F3D_SILENT_WARNING_PUSH()
```

Comment:
Add `F3D_DEPRECATED` as a comment above to not forget to remove that at some point